starlingcaptureapi/actions.py

```diff
@@ -83,12 +88,40 @@ def create_proofmode(self, asset_fullpath, jwt_payload):
         tmp_asset_file = _asset_helper.get_tmp_file_fullpath(".jpg")
         tmp_claim_file = _asset_helper.get_tmp_file_fullpath(".json")
 
-        # TODO: Unzip bundle, store asset file, and create dictionary for claim creation.
         meta_proofmode = None
-
+        # Unzip bundle, store asset file, and create dictionary for claim creation.
+        with tempfile.TemporaryDirectory() as tmp_unzip_folder:
+            with zipfile.ZipFile(asset_fullpath, 'r') as zip_ref:
+                zip_ref.extractall(tmp_unzip_folder)
+                for file in os.listdir(tmp_unzip_folder):
+                    if file.endswith(".jpg"):
+                        shutil.copy2(os.path.join(tmp_unzip_folder, file), tmp_asset_file)
+                        img = Image.open(os.path.join(tmp_unzip_folder, file))
+                        exif_data = img._getexif()
+                        exif_dict = {}
+                        for (k,v) in exif_data.items():
+                            if k in ExifTags.TAGS:
+                                exif_dict[ExifTags.TAGS.get(k)]=v
+
+                        gpsinfo = {}
+                        for key in exif_dict['GPSInfo'].keys():
+                            decode = ExifTags.GPSTAGS.get(key,key)
+                            gpsinfo[decode] = exif_dict['GPSInfo'][key]
+
+                    if file.endswith(".csv"):
+                        csv_reader=csv.DictReader(open(os.path.join(tmp_unzip_folder,file)))
+                        meta_csv = next(csv_reader)
```

Comment:
Is it considered an error / exceptional condition if there are files other than `jpg`/`csv` in the zip? Do we want to log those cases?